pom.xml

```diff
@@ -266,7 +264,6 @@
                 <configuration>
                     <source>${maven.compiler.source}</source>
                     <target>${maven.compiler.target}</target>
-                    <release>${maven.compiler.release}</release>
```

Comment:
Can you check the actual execution logs whether the --release parameter is passed to javac?

Reply:
mvn -X shows
```
[DEBUG] Configuring mojo 'org.apache.maven.plugins:maven-compiler-plugin:3.8.1:compile' with basic configurator -->
[DEBUG]   (f) basedir = /home/jpechane/sources/debezium/debezium/debezium-api
[DEBUG]   (f) buildDirectory = /home/jpechane/sources/debezium/debezium/debezium-api/target
[DEBUG]   (f) compilePath = [/home/jpechane/sources/debezium/debezium/debezium-api/target/classes, /home/jpechane/.m2/repository/org/slf4j/slf4j-api/1.7.30/slf4j-api-1.7.30.jar]
[DEBUG]   (f) compileSourceRoots = [/home/jpechane/sources/debezium/debezium/debezium-api/src/main/java]
[DEBUG]   (f) compilerArgs = [-Xlint:unchecked]
[DEBUG]   (f) compilerId = javac
[DEBUG]   (f) debug = true
[DEBUG]   (f) encoding = UTF-8
[DEBUG]   (f) failOnError = true
[DEBUG]   (f) failOnWarning = false
[DEBUG]   (f) forceJavacCompilerUse = false
[DEBUG]   (f) fork = false
[DEBUG]   (f) generatedSourcesDirectory = /home/jpechane/sources/debezium/debezium/debezium-api/target/generated-sources/annotations
[DEBUG]   (f) mojoExecution = org.apache.maven.plugins:maven-compiler-plugin:3.8.1:compile {execution: default-compile}
[DEBUG]   (f) optimize = false
[DEBUG]   (f) outputDirectory = /home/jpechane/sources/debezium/debezium/debezium-api/target/classes
[DEBUG]   (f) parameters = false
[DEBUG]   (f) project = MavenProject: io.debezium:debezium-api:1.6.0-SNAPSHOT @ /home/jpechane/sources/debezium/debezium/debezium-api/pom.xml
[DEBUG]   (f) projectArtifact = io.debezium:debezium-api:jar:1.6.0-SNAPSHOT
[DEBUG]   (s) release = 8
[DEBUG]   (f) session = org.apache.maven.execution.MavenSession@3c98781a
[DEBUG]   (f) showDeprecation = true
[DEBUG]   (f) showWarnings = true
[DEBUG]   (f) skipMultiThreadWarning = false
[DEBUG]   (f) source = 1.8
[DEBUG]   (f) staleMillis = 0
[DEBUG]   (s) target = 1.8
[DEBUG]   (f) useIncrementalCompilation = true
[DEBUG]   (f) verbose = false
[DEBUG] -- end configuration --
```
Note the `release`

Reply:
Ok, thx for checking!